CompassSDK/Tracker/CompassTracker.swift

```diff
@@ -95,10 +138,15 @@ extension CompassTracker: ScrollPercentProvider {
 }
 
 extension CompassTracker: CompassTracking {
+    @available(*, deprecated, renamed: "setSiteUserId")
     public func setUserId(_ userId: String?) {
-        trackInfo.userId = userId
+        setSiteUserId(userId)
     }
-    
+
+    public func setSiteUserId(_ userId: String?) {
+        trackInfo.siteUserId = userId
+    }
+
     public func getRFV(_ completion: @escaping (String?) -> ()) {
         guard let userId = trackInfo.userId, let accountId = accountId else {
```

Comment:
I'm pretty sure this was wrong before... based on variable names were swapped, this was using siteUserId instead of user id.

```
struct GetRFVApiCall: ApiCall {
    let userId: String
    let account: Int
    let baseUrl: URL?
    let path: String = "data.php"
    var params: [String : Any] {
        ["u": userId, "ac": account]
    }
    
    init(userId: String, account: Int, baseUrl: URL? = Bundle.main.compassEndpoint) {
        self.userId = userId
        self.account = account
        self.baseUrl = baseUrl
    }
}
```